### DISCUSSION TITLE
GitHub home page does not respect prefers-reduced-motion

### DISCUSSION THREAD

**Discussion by CharlesBelov** [community/community#17230] [Accessibility] [answered]
The [GitHub home page](https://github.com/) for not-logged-in users automatically presents an animation. This animation does not respect prefers-reduced-motion. It would be better if a site visitor has prefers-reduced-motion set to display a still frame with a play button.

Unsolicited animation can be distracting to people with attention deficits.

**Comment by benjiallen** [comment]
@CharlesBelov thanks for the feedback. I'll create issues internally and work with the team to understand priority.

Was there a particular animation that you found triggering or does the feedback apply to the whole page?

**Comment by CharlesBelov** [comment]
Specifically, there is a big globe animation which lasts greater than 5 seconds. It interferes with my desire to determine what I want to do next on that page. I'll note that animations over 5 seconds with no pause control are a WCAG 2.0 A 2.2.2 Pause, Stop, Hide violation. That said, it breaks my concentration to receive an unsolicited animation in the first place, which is why I suggested using prefers-reduced-motion to not even serve the animation to begin with to site visitors with that setting.

**Reply by benjiallen** [reply]
Ok, understood. Thanks for the feedback.

**Comment by cehfisher** [comment]
Hello @CharlesBelov - thank you for taking the time to provide this feedback! Our Accessibility team is actively working on this issue. We will follow-up here with any additional questions or updates. Thanks again!

**Comment by cehfisher** [comment, marked as answer]
Great news @CharlesBelov - the animation should now be respecting the user's `prefers-reduced-motion` setting. Could you please retest, keeping in mind you may have to refresh the page to see the change. Thanks!

**Reply by CharlesBelov** [reply]
Confirmed, thank you!
